docs/deprecated-oid-info.md

```diff
@@ -0,0 +1,72 @@
+# Sigstore OID information (deprecated)
```

Comment:
I would prefer that we keep all OIDs in one document, but document which values are deprecated. These will still be needed for verification of all currently issued certificates.

Reply:
Updated 👍